### ISSUE TITLE
*** No rule to make target 'deghost.C', needed by 'deghost.lo'.

### ISSUE THREAD

**Issue by bmckeeby** [robwink/davinci#2]
When running the make command I receive the following error: 

make[2]: Entering directory '/usr/share/davinci'
make[2]: *** No rule to make target 'deghost.C', needed by 'deghost.lo'.  Stop.
make[2]: Leaving directory '/usr/share/davinci'
make[1]: *** [Makefile:822: all-recursive] Error 1
make[1]: Leaving directory '/usr/share/davinci'
make: *** [Makefile:429: all] Error 2

How do I go about correcting this?
system is ubuntu 18.04.02
Thanks!

**Comment by rswinkle** [comment]
I just updated README.md to emphasize that this is a pretty dramatic *fork* of davinci and not the official version.  For the record I did a fresh clone and configured and compiled and it worked but then I figured I might as well try to fix another known bug and it kept saying source file was newer than executable even though ls wasn't showing that.  Running it seemed to confirm that my changes weren't actually being compiled in somehow.

Lots of clean and configure and make no results

I uninstalled davinci from my system (installed using 2.18 deb packages years ago) thinking that might be a problem somehow.

Lots of clean and configure and make and now I get
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_enc_out'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_dec_getsize'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_dec_init'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_dec_free'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_enc_free'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_dec_getimage'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_newlen'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_strerror'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_enc_init'
/home/robert/programming/C/others_code/davinci/davinci/.libs/libdavinci.so: undefined reference to `jbg_dec_in'
collect2: error: ld returned 1 exit status

I've seen that before when I used to work there.  I even remember fixing it.  It's a tiff thing iirc you can google it and it says to link to -ljbig.  Of course this is an autotools project so I pretty much just gave up then because it's not as straight forward as just adding -ljbig.  I loathe autotools and haven't used it since I quit.

This is one of the many reasons davinci should die (see updated README for others).  configuring and cleaning and building over a dozen times across 2 separate repos (a fresh clone and and old one) and getting 3-4 different results including success that doesn't actually seem to pull in changed code ... somehow...?

So long story short, try the official repo.  It's worse in a lot of ways and has a lot more bugs, but it's also missing a few bugs/inconsistencies that I introduced when trying to add the new types and you can always email the mailing list if you have trouble.
